README.md

```diff
@@ -750,7 +750,7 @@ function deployAccount(accountType: AccountImplementationType, options?: DeployA
 
 -   `accountType` - the implementation of the Account that you want to use; currently supported implementations:
     -   `"OpenZeppelin"` - [v0.5.0](https://github.com/OpenZeppelin/cairo-contracts/releases/tag/v0.5.0)
-    -   `"Argent"` - [unofficial v0.2.3](https://github.com/argentlabs/argent-contracts-starknet/tree/3e31c25843010149027ca1bdce251b8d63bdfd9c)
+    -   `"Argent"` - [unofficial v0.2.3](https://github.com/argentlabs/argent-contracts-starknet/tree/780760e4156afe592bb1feff7e769cf279ae9831)
```

Comment:
`unofficial v0.2.3` is not correct anymore. The version introduced with this PR is `unofficial v<whatever comes after v0.2.3>`. Since they didn't increment the version here, not sure what it should be:
https://github.com/argentlabs/argent-contracts-starknet/blob/develop/contracts/account/ArgentAccount.cairo#L37

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/260/commits/6069f840b3e6e74e787da69e47878a756951f653 Named it like: Commit [780760e](https://github.com/argentlabs/argent-contracts-starknet/tree/780760e4156afe592bb1feff7e769cf279ae9831) of branch develop.